sample/playground/src/ormconfig.ts

```diff
@@ -0,0 +1,10 @@
+import "reflect-metadata"
+import { DataSource } from "typeorm"
+import { User } from "./entity/User.js"
+
+export const AppDataSource = new DataSource({
```

Comment:
Can you check if it works with the CLI? `typeorm -d ormconfig.ts migration:generate Test`

I'm sure it works with `export default new DataSource()`, but haven't use it like this. Entity/migration classes work even when there's no default export, but ormconfig probably doesn't use class loader.

Reply:
I did, it works 🙏